### DISCUSSION TITLE
奇忽局04号理事会：卡牌游戏理事会

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#21] [General]
定义和规则有待补充……

@MingxuanXie 曾提到：……[也许资源就是每个（speaker）节点的知识？](https://github.com/qihuju/qihuju.github.io/discussions/9#discussioncomment-544999)节点间的相互链接形成星系（知识体系）及进一步发展成许煜说的宇宙技术

启发：
1. 奇忽局club脑图卡牌：目前每一期奇忽局club都会生成一幅脑图，主要由 @Tz-H 整理，按照 @MingxuanXie 的脑洞，每一幅脑图就是一张“资源卡牌”

- [资源牌4](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-593273)
- [资源牌5](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-593274)
- [资源牌6](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-584331)
- [资源牌7](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-584334)
- [资源牌8](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-584339)
- [资源牌10](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-584344)
- [资源牌11](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-584344)
- [资源牌13](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-584364)

- [资源牌·赛博乡村](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-584340)

- [资源牌·美国众神1&2](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-584344)
- [资源牌·美国众神3](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-584362)

- [资源牌·话题仓库](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-587351)

2. bio角色卡牌：那么每一位speaker的bio就是一张角色卡牌
3. 那么现在就缺少[“技能牌”和“时代牌”了](https://github.com/qihuju/qihuju.github.io/discussions/9#discussioncomment-543914)

··················································
**Clubhouse-mindmap-island”的卡牌游戏** 脑洞「2021年4月10日补充」
受到 @cyberlinden #29 “岛屿|河流二相性”的启发，我觉得可以利用“脑图资源牌”来设计一款《卡坦岛》or《火星殖民》类型的“Clubhouse-mindmap-island”的卡牌游戏：

> 岛屿：就是奇忽局club房间形成的一张一张脑图，类似《卡坦岛》or《火星殖民》的资源牌

这个“脑图岛”其实也可以理解为 [资源牌·话题仓库](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-587351)

> 资源牌：可以做成类似《卡坦岛》or《火星殖民》的六边形模式

> 玩法：基本上按照《卡坦岛》or《火星殖民》的岛屿“殖民”玩法来推进，也呼应了 @Tz-H 和 @midai17 的《奇忽局宣言1.0》的比特荒野殖民的意境 #33 

> NFT：本身现在 @Sleeper2060 也在将奇忽局脑图进行remix，那么如果按照卡牌游戏方向，这些remix出来的“卡牌”也可以NFT化，成为 #25 的new project

**Comment by askender** [comment]
related link: https://github.com/qihuju/qihuju.github.io/discussions/9

**Comment by Lupingyue** [comment]
![奇忽局6：賽博格有性別嗎？——賽博格時代的自我邊界、性別、家庭、婚姻、人際關係](https://user-images.githubusercontent.com/25149468/114019769-6107f480-98a1-11eb-86a9-c98253c9de72.jpg)

**Comment by Lupingyue** [comment]
![奇忽局7：一人一預測，2028超越美國GDP的中國長啥樣？](https://user-images.githubusercontent.com/25149468/114019890-8694fe00-98a1-11eb-9870-dd8fc78e6342.jpg)

**Comment by Lupingyue** [comment]
![奇忽局8：山寨：深圳寨都變成世界創新中心的第一桶金](https://user-images.githubusercontent.com/25149468/114019973-a2989f80-98a1-11eb-9646-4cc5711414dc.jpg)

**Comment by Lupingyue** [comment]
![奇忽局之「賽博鄉村」2：談談Fab_lab在中國，創客運動與鄉村振興，一村一個Fab_lab不是夢？「](https://user-images.githubusercontent.com/25149468/114020096-c956d600-98a1-11eb-9867-78fca6ae295b.jpg)

**Comment by Lupingyue** [comment]
![奇忽局10：“平台合作社”解放平台打工人和快递小哥？「🎤录音房」](https://user-images.githubusercontent.com/25149468/114020159-dc69a600-98a1-11eb-9c4e-7c863ec82c81.jpg)

**Comment by Lupingyue** [comment]
![奇忽局11：跟陈楸帆聊聊AI2041｜Let's_Talk_about_AI2041_with_Che](https://user-images.githubusercontent.com/25149468/114020334-0f139e80-98a2-11eb-8354-570c801d95b7.jpg)

**Reply by Sleeper2060** [reply]
这张脑图打不开。可能需要重新上传。

**Reply by Lupingyue** [reply]
> 这张脑图打不开。可能需要重新上传。

这个要向 @Tz-H 求助，我输出图片几次，都是这样的效果：图片下端是黑的。😢

**Reply by Lupingyue** [reply]
> 这张脑图打不开。可能需要重新上传。

我刚才又补充了两张脑图，这样就够十张了：
- [资源牌4](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-593273)
- [资源牌5](https://github.com/qihuju/qihuju.github.io/discussions/21#discussioncomment-593274)

**Reply by Sleeper2060** [reply]
👌一张两张其实问题不大，哈哈。因为一般需要500张以上的图片来进行train。

**Reply by Lupingyue** [reply]
> 👌一张两张其实问题不大，哈哈。因为一般需要500张以上的图片来进行train。

500张脑图估计需要几年时间😄
不过我的Minecraft作品截图随时可以拿出几千张没有问题😄

**Reply by Tz-H** [reply]
> > 这张脑图打不开。可能需要重新上传。
> 
> 这个要向 @Tz-H 求助，我输出图片几次，都是这样的效果：图片下端是黑的。😢

好像是因为太长的缘故，需要调整脑图里元素的位置（或者隐藏一部分元素）以缩小脑图的高度。

**Comment by Lupingyue** [comment]
![奇忽局9：美國眾神_vs_中國眾仙_1](https://user-images.githubusercontent.com/25149468/114020552-4d10c280-98a2-11eb-829c-3d6ddeeffa94.jpg)

**Comment by Lupingyue** [comment]
![奇忽局之「美国众神vs中国众仙」3：美国土地公_vs_中国耶和华](https://user-images.githubusercontent.com/25149468/114020614-5ef26580-98a2-11eb-98f1-4ecd43c08ec5.jpg)

**Comment by Lupingyue** [comment]
![【奇忽局13】驰骋在比特荒野上的岛屿与河流——漫谈去中心化城市X车联网【🎤录音房】](https://user-images.githubusercontent.com/25149468/114020703-729dcc00-98a2-11eb-8620-6dd4289d1dc3.jpg)

**Comment by Lupingyue** [comment]
![奇忽局：话题仓库](https://user-images.githubusercontent.com/25149468/114115017-98fc4f80-9914-11eb-81be-ca9aedc53094.jpg)

**Comment by Lupingyue** [comment]
![奇忽局4：尋找草薙素子：賽博格的自我與聯網的城市——虛擬身體的多重想象和建構-副本](https://user-images.githubusercontent.com/25149468/114261200-e9a2a400-9a0b-11eb-940f-97cc72deb965.jpg)

**Comment by Lupingyue** [comment]
![奇忽局5：聊聊區塊鏈、NFT、創客、設計師和藝術家的跨界合作，以及實踐中所面臨的法律問題](https://user-images.githubusercontent.com/25149468/114261210-f3c4a280-9a0b-11eb-962a-11f5ebfc7aad.jpg)

**Comment by Sleeper2060** [comment]
脑图的machine learning结果出来了：

部分截图
<img width="834" alt="Screen Shot 2021-04-16 at 6 06 57 PM" src="https://user-images.githubusercontent.com/63529583/115089274-e9107e80-9edf-11eb-9899-bf9e44780921.png">

部分图片动态的演化：

https://user-images.githubusercontent.com/63529583/115089438-3987dc00-9ee0-11eb-85c5-c49ef183dbbd.mov

**Reply by Sleeper2060** [reply]
给的数据不够，所以新造出来的脑图大概在颜色和一些结构上接近。

**Reply by Lupingyue** [reply]
不明觉厉。
能不能解读一下机器学习脑图的逻辑，它学习的是脑图的逻辑和内容？还是图片本身？

**Reply by Sleeper2060** [reply]
这个用的是对图形的学习，预学习模式是自由外型。就是机器已经学习过一些形状。有些是专门学习文字的。
这次的机器学习机制简单点说，就是计算机模拟神经网络运作分成两组进行对抗。
一组模拟，另一组对模拟出来的图形不断的进行判断(就是say yes or no，一般要通过很多次的模拟后才会判别出基本相似）。
像车的自动导航，
就会给机器很多不同的正常路段行驶时候的车的图片，
训练他们对车的外型做出识别。
